site/components/schema/BlogStructuredData.tsx

```diff
@@ -0,0 +1,38 @@
+import { NextSeo } from "next-seo";
+
+export function BlogStructuredData({ blogs }) {
+  const blogStructuredData = blogs.map((blog) => ({
+    "@type": "BlogPosting",
+    headline: blog.title,
+    description: blog.description || "",
+    url: `https://portaljs.com${blog.urlPath}`,
+    datePublished: blog.date,
+    author: (blog.author?.length ? blog.author : ["PortalJS Team"]).map((name) => ({
+      "@type": "Person",
+      name,
+    })),
+  }));
+
+  const jsonLd = {
+    "@context": "https://schema.org",
+    "@type": "Blog",
```

Comment:
@Datopian-Shreyas do we think that index page for blog posts is a blog post itself? I'm not really sure if so.